frontend/config.json

```diff
@@ -1,49 +1,4 @@
 {
```

Comment:
Please remove this file from git and add it to gitignore.

Reply:
frontend/config.json in gitignore file

Reply:
removed file